record_indexer/README.md

```diff
@@ -22,6 +22,8 @@ Add the current stage index for a collection to the `rikolti-prd` alias:
 python -m record_indexer.move_index_to_prod <collection_id>
 ```
 
+Note that the index creation code enforces the existence of one stage index at a time. This means we can simply supply the collection ID as input and the process will move the current stage index to production.
```

Comment:
Could you elaborate on the lifecycle of an index? I know we've discussed all this but I can't quite remember the rules. 

We create a new index with a stage alias each time we run the harvesting pipeline, presumably also deleting any existing indices with a stage alias? 

At some point we're happy with a stage index and we want to push it to production, so we run the publishing dag, which adds the prod alias to the index, and removes the stage alias? so now we have no indices with a stage alias but one index with a prod alias? 

At some point we repeat this process, and when we add the prod alias to the new index that's ready for publishing, then the index with the prod alias becomes a plain unaliased index? these unaliased indices are kept around per the retention policy sorted by date? so as a rule, any index without an alias is a previously published index? and, as a rule, if that index retention policy were set to more than 1, we could sort any index without an alias by date to, for example, restore the most recently published index?

Reply:
README is updated.